SCSS/style.scss

```diff
@@ -0,0 +1,992 @@
+                p  {
+                width: 100%;
+              
+              }
+        }
+            .banner-btn{
+                color: white;
+                background-color: brown;
+            }
+            
+        }
+        @media only screen and (max-width:758px) {
+            .inner-div  {
+            width: 80%;
+          
+          }
+    }
+        
+    }
+
+    // banner slider ends 
+    
+   
+}
+@media only screen and (max-width:768px){
+    .banner{
+        height: 70vh;
+        .banner-div{
+            .slick-slider{
+                .slick-dots{
+                    bottom: -80px;
+                    button::before{
+                        color: white;
+                    }
+                }
+                
+           }
+            .inner-div{
+                width: 80%;
+                margin: 10vh auto;            
+                text-align: center;
+                color: white;
+                align-content: center;
+                p{
+                    width: 80%;
+                }
+            }
+        }
+        
+    }
+}
+@media only screen and (max-width:576px){
+    .banner{
+        height: 60vh;
+        .banner-div{
+            .slick-slider{
+                .slick-dots{
+                    bottom: -70px;
+                    button::before{
+                        color: white;
+                    }
+                }
+                
+           }
+            .inner-div{
+                width: 100%;
+                margin: 10vh auto;            
+                text-align: center;
+                color: white;
+                align-content: center;
+                p{
+                    width: 100%;
+                }
+            }
+        }
+        
+    }
+}
+ // feature starts here 
+ .feature{
+    .f-row{
+        margin-top: 55px;
+        text-align: center;
+        margin-bottom: 55px;
+    }
+    .f-col{
+        i{
+            font-size: 40px;
+            &::before{
+                margin-bottom: 20px;
+            }
+        }
+        p{
+            margin-top: 10px;
+            font-size: $font-para;
+            color: rgb(155, 162, 168);
+        }
+        .line{
+            color: lightgray;
+           font-size:$font-heading;
+           font-weight: 800;
+           margin: 0;
+        }
+    }
+    .f-col:hover *{
+        color: $theme-text-color;
+    }
+ }
+ // feature ends here
+
+ //design-div3 starts here
+ 
+.designs{
+    margin: 0 auto;
+    text-align: center;
+    background-color: rgb(255, 255, 244);
+    padding-top: 100px;
+    .div3-heading{
+        text-align: center;
+    }
+    .div3-para{
+        width: 60%;
+        text-align: center;
+        margin: 0 auto;
+    }
+    .div4-slick-slider {
+        margin: 0 auto;
+        padding: 0px;
+        .slick-track{
+            padding: 102px 0;
+            bottom: 0;
+        }
+        @media only screen and (max-width: 1400px){
```

Comment:
this is not the perfect way to use media query please fix this

Reply:
done